src/BridgeFactory.sol

```diff
@@ -0,0 +1,101 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.25;
+
+import {IERC1271} from "./interfaces/IERC1271.sol";
+import {IERC20} from "@forge-std/interfaces/IERC20.sol";
+import {TwoStepOwnable} from "./deployer/TwoStepOwnable.sol";
+import {SafeTransferLib} from "./vendor/SafeTransferLib.sol";
+import {MerkleProofLib} from "./vendor/MerkleProofLib.sol";
+import {Context} from "./Context.sol";
+
+contract BridgeFactory is IERC1271, TwoStepOwnable, Context {
```

Comment:
`Context` always has to go first after interfaces. You get some _very_ strange and wrong behavior if it comes late in the MRO